Library/Homebrew/cmd/untap.rb

```diff
@@ -5,7 +5,7 @@ def untap
     raise "Usage is `brew untap <tap-name>`" if ARGV.empty?
 
     ARGV.each do |tapname|
-      user, repo = tap_args(tapname)
+      user, repo = tap_args()
```

Comment:
That's true. I'll need to think about that more. (Does that explain why you don't want me to change the `tap_args` method?)